flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/security/token/HiveServer2DelegationTokenProviderITCase.java

```diff
@@ -49,15 +49,15 @@
 public class HiveServer2DelegationTokenProviderITCase {
```

Comment:
```suggestion
class HiveServer2DelegationTokenProviderITCase {
```

Note: the public can be removed for all test methods.